compiler/src/main/kotlin/dev/zacsweers/metro/compiler/ir/Binding.kt

```diff
@@ -463,11 +465,20 @@ internal sealed interface Binding : BaseBinding<IrType, IrTypeKey, IrContextualT
         multibinds: IrAnnotation,
         contextualTypeKey: IrContextualTypeKey,
       ): Multibinding {
+        val isDaggerMultibinds =
+          multibinds.ir.symbol.owner.parentClassId?.asSingleFqName() ==
+            FqName("dagger.multibindings.Multibinds")
```

Comment:
I saw `Symbols.DaggerSymbols.ClassIds`, let me know if it's better to move the declaration there